### ISSUE TITLE
Customize UI option to toggle the extra information in brackets on or off in the available main game list

### ISSUE THREAD

**Issue by neosquid** [mamedev/mame#2079]
A Customize UI option to toggle the extra information in brackets on or off in the available main game list leaving only the game name showing:

e.g. "Capcom Sports Club (US)" to show simply the game name "Capcom Sports Club"

This would make the front end look tidier than having all this extra information cluttering the game list.

**Comment by Tafoid** [comment]
That would not be an impossible task.  On a Front-End I made one could use a keyboard shortcut (for me it was CTRL key combo) which simply set a variable/flag it so when the routine to display descriptions on screen was hit in code, if toggled, instead of just printing the full string it would print a trimmed string of only the text that shows up before the "**(**" character if present.

In any case, while I could see how one could consider the extra version/region information in a description "clutter", it is useful information the most people use to determine what machine to choose.

**Comment by LnmVolbo** [comment]
@ **neosquid**,

Seems to me that if all the stuff in parenthesis were hidden, the result would be a list with lot of entries that looked like duplicates.  Perhaps reducing the clutter, but causing a worse (in my opinion) problem - a confusing user interface.

Perhaps you meant to only eliminate the entries that were still unique once the parenthetical information was hidden?

Because there are cases like this:
   Capcom Sports Club (Euro 971017)
   Capcom Sports Club (Euro 970722)
   Capcom Sports Club (Euro 970722 Phoenix Edition) (bootleg)
   Capcom Sports Club (Asia 970722)
   Capcom Sports Club (Hispanic 970722)
   Capcom Sports Club (Japan 970722)
   Capcom Sports Club (Japan 970722, yellow case)

So the seven variants of _Capcom Sports Club_ would still appear in their entirety, but a set like _Capcom Baseball (Japan)_ would appear as _Capcom Baseball_.

But let's look at the numbers first.  If you use the -listfull command you can make a list of all the descriptions in MAME.  It is approx. 34050 items long.  Of those, approx. 29900 of those have parenthesis in the description.  Obviously a lot of clutter, eh?  Problem is, only about 4170 of the 29900 descriptions are unique once the parentheses are removed.  (Yes, I imported the list into a spreadsheet and made a count - I approximated some numbers because I used a MAME WIP version and not an official version.)

So, not practical.  Even if one went to the trouble to remove the parentheses on non-unique descriptions, around 86% would still be appear cluttered, according to your "parenthesis" bias.

I agree with **Tafoid**.  Many, if not most of, MAME users need to see the stuff in parenthesis to make an exact selection.

Perhaps your idea would make more sense if it only functioned while viewing the list through one of the filters, like "parents".  There are around 22912 clones, if they were removed from the list of 34050, the percentage of "unique descriptions after the parentheses are removed" should be much better than the ~14% I tabulated previously.

If you remove the clones from the list, it leaves around 8078 having parenthesis.  And when you check for unique descriptions once the parenthesis are removed, the result is 6267.  Now your idea makes some sense, as the ratio of "de-clutterable" descriptions is nearly 78%.

Summing up, removing any of the parenthetical info on the full list = bad idea.  Removing _some_ of the parenthetical info in the parents list = good idea.

And by the way, there is no MAME game with the description "Capcom Sports Club (US)".

**Comment by neosquid** [comment]
<game name="10yard85" index="" image="">
		<description>10-Yard Fight &apos;85 (US, Taito license)</description>
		<shortdescription>10-Yard Fight</shortdescription >
		<cloneof>10yard</cloneof>
		<crc></crc>
		<manufacturer>Irem (Taito license)</manufacturer>
		<year>1985</year>
		<genre>Sports / Amer. Football</genre>
		<rating>AAMA - Green (Suitable For All Ages)</rating>
		<enabled>Yes</enabled>
	</game>

If for example, <shortdescription> is added to each parent and clone entry in the xml/database that MAME uses; then if the option is toggled, the data from that tag is listed in the MAME menu you are looking at. Duplicates aren't an issue for me as I don't keep multiple roms of the parent rom, and anyway, the option can be selected to show the list with the full info when needed.

Probably, most people would like to see the full information. Myself from an end user perspective would like the choice of an aesthetically pleasing menu. It would be an advanced option not default.

Thanks for your time to consider and reply to my initial request.

**Comment by neosquid** [comment]
How about only "favourites" section has the parenthesis information hidden?

That would be a nice compromise.

**Comment by LnmVolbo** [comment]
In view of **neosquid's** reply on Feb 22nd, shouldn't the title be changed to _UI Request to Customize UI option to toggle the extra information in brackets on or off in the **favorite's** game list_?

It was determined to be impractical to do so on the available main list.

**Comment by neosquid** [comment]
Revisiting MAME tonight using the UI, had the idea that the full game name should be listed in the lower green information area (which also should be able to be hidden by choice, as the side panels are). This avoids any confusion as some have argued, who want to make the exact selection.
